searchlib/src/main/java/com/yahoo/searchlib/rankingexpression/evaluation/FeatureNames.java

```diff
@@ -0,0 +1,76 @@
+package com.yahoo.searchlib.rankingexpression.evaluation;// Copyright 2018 Yahoo Holdings. Licensed under the terms of the Apache 2.0 license. See LICENSE in the project root.
```

Comment:
Well, I guess this does count as having the license header on the first line....